CreateProject.sql

```diff
@@ -0,0 +1,26 @@
+DELIMITER $$
+DROP PROCEDURE IF EXISTS createProjectTable $$
+CREATE PROCEDURE createProjectTable()
+BEGIN
+    START TRANSACTION;
+		CREATE TABLE Projects (
+		ProjectID SMALLINT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+		ProjectName VARCHAR(50) NOT NULL,
+		ProjectValue DECIMAL(15, 2) NOT NULL,
+        ProjectTechLeadID SMALLINT UNSIGNED NOT NULL
+		);
+
+		CREATE TABLE ProjectTechnologies (
```

Comment:
You should have a Technology table, which is ID and Name. The Project Technology table should then be a Composite Primary Key of Project ID and Technology ID